check-envs.sh

```diff
@@ -13,6 +13,16 @@ if [ ! "${POT_MOUNT_BASE}" ] ; then
 	POT_MOUNT_BASE=/opt/pot/
 	echo POT_MOUNT_BASE not set, using ${POT_MOUNT_BASE}
 fi
+if [ -z "$GITHUB_TOKEN" ] || [ -z "$GITHUB_ORG" ]; then
```

Comment:
Rather than checking the same condition twice, it might be cleaner to check whether the target exists and then whether some variable was correctly sourced from it. Since `/root/.profile` is guaranteed to be present when the scripts are run, the outside block isn't really necessary at all.

```
if [ -f "/root/.profile" ]; then
	. /root/.profile
	# Check to confirm the variables are set
	if [ -z "$GITHUB_TOKEN" ] || [ -z "$GITHUB_ORG" ]; then
		echo "Error: Environment variables could not be sourced from /root/.profile."
	fi
fi
```